### ISSUE TITLE
Adding class weights and training with resnet 18

### ISSUE THREAD

**Issue by nithinme3** [google-research/deeplab2#101]
Hi,

I wanted to add class weights to the existing code according to number of total pixels each classes have, so that the training results will be improved. So where I need to modify the codeto incorporate this?
Also I could find pretrained network starting from resnet50 from the documentation provided. Will I be able to train with resnet18 with the existing repository?

**Comment by aquariusjay** [comment]
Hi @nithinme3,

Please check the losses implemented in [base_loss.py](https://github.com/google-research/deeplab2/blob/main/model/loss/base_loss.py).
We don't have ResNet-18 ImageNet pretrained checkpoint at hand. If you would like to try small model variants, maybe you could take a look at [MobileNetv3-{small,large}](https://github.com/google-research/deeplab2/blob/main/g3doc/projects/imagenet_pretrained_checkpoints.md#checkpoints).

Cheers,

**Comment by aquariusjay** [comment]
Closing the issue, since there is no active discussion for a long time.
Please feel free to reopen it if needed.
